src/config.ts

```diff
@@ -12,11 +12,12 @@ export const SIDEBAR = {
 		{ text: 'Components', link: 'en/core-concepts/astro-components' },
 		{ text: 'Pages', link: 'en/core-concepts/astro-pages' },
 		{ text: 'Layouts', link: 'en/core-concepts/layouts' },
-		{ text: 'Routing', link: 'en/core-concepts/routing' },
-		{ text: 'Partial Hydration', link: 'en/core-concepts/component-hydration' },
+		{ text: 'Framework Components', link: 'en/core-concepts/framework-components' },
```

Comment:
Possibly. Thinking on this!  :)

Reply:
I will let @FredKSchott make the final call here. I don't strongly prefer either option:

In favour of just Components: 
- it's shorter! 
- nothing else says Astro pages , Astro layouts etc
- it's probably *assumed* Astro, being in and among the other things
- it's emphasized as being the "default" by not having an adjective

In favour of Astro Components
- it says Astro again! Better SEO, Better visibility
- people in their head might be using the phrase "Astro component", so this might match what they have in mind when they go looking in the docs for help
- it teaches users to distinguish between what KIND of components their using, especially when asking for support help
- if people go looking for help with e.g. React components, and see "Components" first, or without noticing Framework components, then they might assume they go there for "component" help/info and it will take them longer to get to Framework components.

Reply:
Ah interesting! I just changed it from Astro Components -> Components so that it matched "Components / Layouts / Themes" I'm not really sure now! 

Maybe "Framework Components" is the odd one now. I know this sounds silly, but maybe we can play around with something more like "React, Svelte, Vue, etc."

Reply:
OK, I'll think on this. Maybe even just "Frameworks"?  (We do use "bring your favourite Framework" often, so I think there's context for that.  (No need to change the url from framework-components, I don't think. Just thinking about the sidebar wording.)

Reply:
This could be a good poll for our community, post-merge.

Reply:
I'll consider "Frameworks" to be our solution until we like something better!